pkg/store/copr/batch_coprocessor.go

```diff
@@ -919,17 +928,16 @@ func buildBatchCopTasksCore(bo *backoff.Backoffer, store *kvStore, rangesForEach
 			}
 		}
 
-		var batchTasks []*batchCopTask
-		var regionIDsInOtherZones []uint64
-		var regionInfosNeedReloadOnSendFail []RegionInfo
-		storeTaskMap := make(map[string]*batchCopTask)
+		rpcCtxs := make([]*tikv.RPCContext, 0)
+		validTiflashStores := make([][]uint64, 0)
+		allValidTiflashStores := make(map[uint64]struct{}, 0)
```

Comment:
The names of `validTiFlashStores` and `allValidTiFlashStores` make me confused. How about rename it to `usedTiFlashStores` and `usedTiFlashStoresMap`

Reply:
> The names of `validTiFlashStores` and `allValidTiFlashStores` make me confused. How about rename it to `usedTiFlashStores` and `usedTiFlashStoresMap`

okk